continuous_integration/scripts/install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -vxeuo pipefail
+
+# These packages are installed in the base environment but may be older
+# versions. Explicitly upgrade them because they often create
+# installation problems if out of date.
+python -m pip install --upgrade pip setuptools wheel numpy
+# Versioneer uses the most recent git tag to generate __version__, which appears
+# in the published documentation.
+git fetch --tags
+python -m pip install .
```

Comment:
Optional suggestion for a more verbose installation the same way as it's done in our conda feedstocks. Feel free to ignore.
```suggestion
python -m pip install . -v
```